src/components/LoadingDialog/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface LoadingDialogProps {
+  'data-test-id'?: string;
+}
+
+const LoadingDialog = (htmlAttributes: LoadingDialogProps): JSX.Element => {
+  return (
+    <div className="w-full h-full bg-black bg-opacity-60 fixed top-0 left-0 right-0" {...htmlAttributes}>
```

Comment:
We don't need that, right?
```suggestion
    <div className="w-full h-full bg-black bg-opacity-60 fixed top-0 right-0" {...htmlAttributes}>
```